app/src/main/java/org/mozilla/focus/browser/HomeTileRecyclerView.kt

```diff
@@ -65,18 +66,21 @@ class HomeTileAdapter(
             TelemetryWrapper.homeTileClickEvent(it.context, item)
         }
 
+        itemView.setOnLongClickListener {
+            openContextMenu?.invoke()
```

Comment:
nit: `openContextMenu` -> `onTileLongClick`

To decouple a hierarchy of objects, the child shouldn't know what the parent is going to do in the callback so we name it generically. If we call it something like `openContextMenu`:
- The name needs to change if the behavior changes, or this child gets a new parent
- Having an `openContextMenu` may encourage someone to create another callback to have another action occur, e.g. `displayToast`, even if they both occur in the long click listener.